app/views/observer_mailer/proposal_complete.html.haml

```diff
@@ -2,7 +2,7 @@
 - top_head = t("mailer.observer_mailer.proposal_complete.header")
 - proposal_link_text = t("mailer.view_request_cta")
 - cta_subheader = t("mailer.proposal_status_html",
-  proposal_status: @proposal.detailed_status)
+  proposal_status: @proposal.detailed_status.capitalize)
```

Comment:
maybe we should call `capitalize` in the `detailed_status` method itself so we don't have to do it each time we call the method?